rotkehlchen/premium/premium.py

```diff
@@ -139,6 +140,7 @@ def reset_credentials(self, credentials: PremiumCredentials) -> None:
         self.credentials = credentials
         self.session.headers.update({
             'API-KEY': self.credentials.serialize_key(),
+            'User-Agent': f'rotki/{get_system_spec()["rotkehlchen"]}',
```

Comment:
dude ... 
```suggestion
            'User-Agent': f'rotki/{get_system_spec()['rotkehlchen']}',
```

Reply:
@LefterisJP  I used double quotes because you are formatting the string with single quotes. Is `""` inside `''`. If I used `''` It would be a syntax error. Also this is the format that I've seen in other parts of the code. The other option is `''` inside `""`

Reply:
Lol you are right. I missed it's inside an f string.